src/Orleans.Core/Runtime/InvokableObjectManager.cs

```diff
@@ -170,9 +189,17 @@ private async Task LocalObjectMessagePumpAsync(LocalObjectData objectData)
                     else if (message.Direction != Message.Directions.OneWay)
                         this.SendResponseAsync(message, resultObject);
```

Comment:
why is this method name SendResponseAsyc, it is not async method though.

Reply:
Maybe because it's non-blocking, but I'm not sure. It could be renamed at some point.